RecoTracker/LSTCore/src/alpaka/LSTEvent.dev.cc

```diff
@@ -418,10 +412,7 @@ void LSTEvent::createSegmentsWithModuleMap() {
     alpaka::memset(queue_, totOccupancySegments_view, 0u);
   }
 
-  Vec3D const threadsPerBlockCreateSeg{1, 1, 64};
-  Vec3D const blocksPerGridCreateSeg{1, 1, nLowerModules_};
-  WorkDiv3D const createSegments_workDiv =
-      createWorkDiv(blocksPerGridCreateSeg, threadsPerBlockCreateSeg, elementsPerThread);
+  auto const createSegments_workDiv = cms::alpakatools::make_workdiv<Acc3D>({nLowerModules_, 1, 1}, {1, 1, 64});
```

Comment:
I haven't looked much at this PR, but is changing this from {1,1,nLower} to {nLower,1,1} intentional?